spec/features/user_can_be_deleted_spec.rb

```diff
@@ -6,7 +6,8 @@
 
     it 'can destroy user' do
       user = create(:default_user)
-      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+      allow_any_instance_of(ApplicationController).to receive(:current_user)
+                                                  .and_return(user)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 44) spaces for indenting an expression spanning multiple lines.